bin/s2nc.c

```diff
@@ -723,6 +754,16 @@ int main(int argc, char *const *argv)
             s2n_connection_set_dynamic_record_threshold(conn, dyn_rec_threshold, dyn_rec_timeout);
         }
 
+        if (serialize_out) {
```

Comment:
Is this the right place to do this? What if you then echo data? The serialized connection won't be right. The sequence number will be wrong.

I guess it'll work for how you want to use it because both sides will be wrong in the same way, but if only one side serializes/deserializes then it'll fail.